### DISCUSSION TITLE
Experiencing difference between local site and github.io page

### DISCUSSION THREAD

**Discussion by d5nguyen** [community/community#22574] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>I am seeing some differences between my local site and what I see when I click on my <a href="http://github.io" rel="nofollow noopener">github.io</a> page. </p>
<p>Here’s the local site:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/2666501acb46abd794a6750f8f59e86dc1cdba76.png" data-download-href="/uploads/short-url/5tHlALhmhGFR1lX6XqMeqZEmxds.png?dl=1" title="Screen Shot 2018-11-26 at 8.58.15 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33303347/181072031-09517044-396f-45e3-8b52-4293d10bf92d.png" alt="Screen Shot 2018-11-26 at 8.58.15 PM.png" data-base62-sha1="5tHlALhmhGFR1lX6XqMeqZEmxds" width="690" height="326" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/2666501acb46abd794a6750f8f59e86dc1cdba76_2_690x326.png, https://aws1.discourse-cdn.com/github/optimized/1X/2666501acb46abd794a6750f8f59e86dc1cdba76_2_1035x489.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/2666501acb46abd794a6750f8f59e86dc1cdba76.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/2666501acb46abd794a6750f8f59e86dc1cdba76_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-11-26 at 8.58.15 PM.png</span><span class="informations">1115×527 23.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>and here is how it looks when I access it via &lt;<a href="https://d5nguyen.github.io/whatsfordinah/Contact.html" rel="nofollow noopener">https://d5nguyen.github.io/whatsfordinah/Contact.html</a>&gt;:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/8f7c150c5cb9cc50b11d166a512ebfe62293eb39.png" data-download-href="/uploads/short-url/ktk7DB1npq1xfsKbuGyKSleAdqp.png?dl=1" title="Screen Shot 2018-11-26 at 8.58.42 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33303347/181072033-e20a1fbe-8837-4650-af9a-92f2127f3eb6.png" alt="Screen Shot 2018-11-26 at 8.58.42 PM.png" data-base62-sha1="ktk7DB1npq1xfsKbuGyKSleAdqp" width="689" height="258" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/8f7c150c5cb9cc50b11d166a512ebfe62293eb39_2_689x258.png, https://aws1.discourse-cdn.com/github/optimized/1X/8f7c150c5cb9cc50b11d166a512ebfe62293eb39_2_1033x387.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/8f7c150c5cb9cc50b11d166a512ebfe62293eb39.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/8f7c150c5cb9cc50b11d166a512ebfe62293eb39_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-11-26 at 8.58.42 PM.png</span><span class="informations">1231×461 22.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Can someone kindly explain to me what is going on? I have looked into Jekyll but it appears it’s used primarily for Ruby? and I am only using basic HTML and CSS at the moment. </p>
<p>Thanks in advance for your help! Here’s the link to my repository: <a href="https://github.com/d5nguyen/whatsfordinah" rel="nofollow noopener">https://github.com/d5nguyen/whatsfordinah</a></p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/d5nguyen">@d5nguyen</a>,</p>
<p>Thanks for being here! Would you mind sharing a link to your repo so we can take a look?.</p>
<p>Cheers,</p>
<p>Andrea</p>

**Comment by d5nguyen** [comment]
<p>thanks for the reminder! I’ve added it to my original message :slight_smile:</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/d5nguyen">@d5nguyen</a>,</p>
<p>I’m not certain why there’s a difference between your local site and the repository unless you pushed up a change that triggered a build error, but it looks like your <strong>&lt;</strong>** header&gt;** tag has “display: flex” on it which makes the style difference between your local site and your site on GitHub Pages.</p>
<p>Hope that helps you track down the problem! If I can help more, please let me know. Thanks!</p>

**Comment by d5nguyen** [comment]
<p>hey thanks, that-pat for your response. I had commented that code out since at the time I wanted to save it in case I needed it so I don’t think it was really affecting the display. I’ve since deleted that code and for some reason the issue remains…still unclear</p>

**Comment by d5nguyen** [comment, marked as answer]
<p>I figured it out!! I wrapped my form with a div that was assigned as a “row” class and that made it all wonky. I copied that form from a previous project and so inherited some of the quirks. </p>
<p>Thank you again for your contribution!!</p>

**Comment by ThirdEyeRose** [comment]
<p>You’re very welcome! I’m glad that you got it sorted. :slight_smile:</p>
